packages/web-react/src/components/TextArea/README.md

```diff
@@ -0,0 +1,47 @@
+# TextArea
+
+TextArea enables the user to type in text information. It has textarea, label,
+and an optional message. It could be disabled or have an error state. The label could be hidden
+and show if the textarea is required.
+
+```jsx
+<TextArea id="example" type="text" name="example" isRequired validationState="error" message="validation failed" />
+<TextArea id="example" type="password" name="example" isRequired validationState="error" message="validation failed" />
+<TextArea id="example" type="email" name="example" isRequired validationState="error" message="validation failed" />
+```
+
+## Available props
+
+| Name              | Type                          | Description                       |
+| ----------------- | ----------------------------- | --------------------------------- |
+| `id`              | string                        | Textarea and label identification |
+| `name`            | string                        | Textarea name                     |
+| `label`           | string                        | Label text                        |
+| `placeholder`     | string                        | Textarea placeholder              |
+| `value`           | string                        | Textarea value                    |
+| `maxLength`       | number                        | Maximum number of characters      |
+| `message`         | string                        | Validation or help message        |
+| `rows`            | number                        | Number of visible rows            |
+| `validationState` | `success`, `warning`, `error` | Type of validation state          |
+| `isDisabled`      | boolean                       | Whether is field disabled         |
+| `isRequired`      | boolean                       | Whether is field required         |
+| `isLabelHidden`   | boolean                       | Whether is label hidden           |
+
+## Custom component
+
+Text field classes are fabricated using `useTextAreaStyleProps` hook. You can use it to create your own custom TextArea component.
+
+```jsx
+const CustomTextArea = (props: SpiritTextAreaProps): JSX.Element => {
+  const { classProps, props: modifiedProps } = useTextAreaStyleProps(props);
+
+  return (
+    <div className={classProps.root}>
+      <textarea {...modifiedProps} className={classProps.input}></textarea>
+      <label className={styleProps.label}>{props.label}</label>
```

Comment:
`label` is missing `for` attribute.

Reply:
Thanks. It was missing in the documentation for every form component. 👍🏻